README.md

```diff
@@ -8,49 +9,56 @@ This repository also contains the Zowe Node Client SDK. The SDK lets you leverag
-| General guidelines that apply to contributing to Zowe CLI and Plug-ins | [Contribution Guidelines](./CONTRIBUTING.md) |
-| Conventions and best practices for creating packages and plug-ins for Zowe CLI | [Package and Plug-in Guidelines](./docs/PackagesAndPluginGuidelines.md)|
-Guidelines for contributing to Zowe SDKs| [SDK Guidelines](./docs/SDKGuidelines.md) |
-| Guidelines for running tests on Zowe CLI | [Testing Guidelines](./docs/TESTING.md) |
-| Guidelines for running tests on the plug-ins that you build| [Plug-in Testing Guidelines](./docs/PluginTESTINGGuidelines.md) |
+| General guidelines that apply to contributing to Zowe CLI and Plug-ins | [Contribution guidelines](./CONTRIBUTING.md) |
+| Conventions and best practices for creating packages and plug-ins for Zowe CLI | [Package and plug-in guidelines](./docs/PackagesAndPluginGuidelines.md)|
+Guidelines for contributing to Zowe SDKs| [SDK guidelines](./docs/SDKGuidelines.md) |
+| Guidelines for running tests on Zowe CLI | [Testing guidelines](./docs/TESTING.md) |
+| Guidelines for running tests on the plug-ins that you build| [Plug-in testing guidelines](./docs/PluginTESTINGGuidelines.md) |
 | Documentation that describes the features of the Imperative CLI Framework | [About Imperative CLI Framework](https://github.com/zowe/imperative/wiki) |
-| Naming CLI commands and developing syntax | [Command Format Standards](./docs/CommandFormatStandards.md) |
-Versioning conventions for Zowe CLI and Plug-ins| [Versioning Guidelines](./docs/MaintainerVersioning.md) |
-| Miscellaneous tips for development | [Development Tips](./docs/DevelopmentTips.md)
+| Naming CLI commands and developing syntax | [Command format standards](./docs/CommandFormatStandards.md) |
+Versioning conventions for Zowe CLI and Plug-ins| [Versioning guidelines](./docs/MaintainerVersioning.md) |
+| Miscellaneous tips for development | [Development tips](./docs/DevelopmentTips.md)
 
-**Tip:** 
-- Visit our [Sample Plug-in repository](https://github.com/zowe/zowe-cli-sample-plugin) for example plug-in code. You can follow developer tutorials [here](https://docs.zowe.org/stable/extend/extend-cli/cli-devTutorials.html).
+**Tip:** Visit our [Sample plug-in repository](https://github.com/zowe/zowe-cli-sample-plugin) for example plug-in code. Follow the [developer tutorials](https://docs.zowe.org/stable/extend/extend-cli/cli-devTutorials.html) for more tips.
 
 <br/>
 
-## **Building Zowe CLI From Source**
-Zowe CLI requires NPM version 8 and Cargo version 1.72.0 (or newer) to build from source. Before proceeding, check your NPM version with `npm --version` and if it's older than 8.x, update with `npm install -g npm`. To check your version of Cargo, run `cargo --version`. Cargo can be installed using rustup: [https://rustup.rs/](https://rustup.rs/). To update Cargo, run the `rustup update` command.
+## Building Zowe CLI from source
+
+Zowe CLI requires NPM version 8 and Cargo version 1.72.0 (or newer) to build from source.
```

Comment:
Should we say `NPM version corresponding to (or bundled with) the active LTS versions of NodeJS`?

That way we don't have to update this every other year 😋